### PR TITLE
feat(arch): move && simplify component skeleton

### PR DESCRIPTION
**Pull request by feoche**
:warning: BREAKING CHANGES :warning: 

    OsdsSkeleton is no longer part of @ovhcloud/ods-stencil package
    OsdsSkeleton is now accessible by adding either ods-component-skeleton or ods-components dependencies
    @ovhcloud/ods-stencil-skeleton lib has been replaced by @ovhcloud/ods-component-skeleton
    @ovhcloud/ods-stencil-skeleton-react lib has been replaced by @ovhcloud/ods-component-skeleton-react
    @ovhcloud/ods-stencil-skeleton-vue lib has been replaced by @ovhcloud/ods-component-skeleton- vue
    OdsSkeletonSize has been removed from @ovhcloud/ods-core and is replaced by ODS_SKELETON_SIZE from @ovhcloud/ods-component-skeleton
    OdsSkeletonSizeList has been removed from @ovhcloud/ods-core and is replaced by ODS_SKELETON_SIZES from @ovhcloud/ods-component-skeleton
    OdsSkeletonSizeUnion has been removed from @ovhcloud/ods-core
    @ovhcloud/ods-component-skeleton exports only OdsSkeletonAttribute as component interface.

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-storybook#1191.0 ✘  
* 
  * build ✘

File: packages-new/components/skeleton/src/components/osds-skeleton/constants/skeleton-size.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+enum ODS_SKELETON_SIZE {
+  xs = 'xs',
+  sm = 'sm',
+  md = 'md',
+  lg = 'lg',
+  xl = 'xl',
```
Comment (dpellier) [inline packages-new/components/skeleton/src/components/osds-skeleton/constants/skeleton-size.ts:6]: alpha order would be nice

File: packages-new/components/skeleton/src/components/osds-skeleton/interfaces/attributes.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { ODS_SKELETON_SIZE } from '../constants/skeleton-size';
```
Comment (dpellier) [inline packages-new/components/skeleton/src/components/osds-skeleton/interfaces/attributes.ts:1]: type

File: packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import type { OdsSkeletonAttribute } from './interfaces/attributes';
+import { Component, Element, h, Host, Prop } from '@stencil/core';
+import { HTMLStencilElement } from '@stencil/core/internal';
```
Comment (dpellier) [inline packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx:3]: type

File: packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import { ODS_SKELETON_SIZE } from './constants/skeleton-size';
```
Comment (dpellier) [inline packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx:4]: type

File: packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import { DEFAULT_ATTRIBUTE } from './constants/default-attributes';
+
+@Component({
+  tag: 'osds-skeleton',
+  styleUrl: 'osds-skeleton.scss',
+  shadow: true
+})
+export class OsdsSkeleton implements OdsSkeletonAttribute {
+  /** @see OdsSkeletonBehavior.hostElement */
+  @Element() hostElement!: HTMLStencilElement;
```
Comment (dpellier) [inline packages-new/components/skeleton/src/components/osds-skeleton/osds-skeleton.tsx:14]: probably not needed

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by dpellier — APPROVED** [review]

**Pull request merged**